src/routes/Systems/components/SystemsTable/constants.js

```diff
@@ -1,6 +1,17 @@
+// TODO Move serialisers.js within SystemsTable directory
```

Comment:
We pass the default `items` function to fetch systems from the Inventory API in the `SystemsTable`, so this should also be the place where we pass the default serialisers for it.

I didn't move the file yet, but we should at some point.